core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -132,9 +132,11 @@ class TaskManager(taskManagerResources: SystemResources = TaskManagerDefaults.de
                   logDirectory: Option[Path]                 = None,
                   scheduler: Scheduler                       = TaskManagerDefaults.defaultScheduler,
                   simulate: Boolean                          = false,
-                  sleepMilliseconds: Int                     = 10
+                  sleepMilliseconds: Int                     = 250
 ) extends TaskManagerLike with TaskTracker with FinalStatusReporter with LazyLogging {
 
+  private var curSleepMilliseconds: Int = sleepMilliseconds
```

Comment:
Perhaps we should just take this out of the constructor altogether, initialize it to something fairly low (e.g. 50ms) and then let these changes take care of optimizing it?